poetry_multiproject_plugin/components/parsing/rewrite.py

```diff
@@ -35,6 +36,7 @@ def mutate_import_from(
 
 
 def mutate_imports(node: ast.AST, namespaces: List[str], top_ns: str) -> bool:
+    ast.FormattedValue
```

Comment:
I'm curious of what this do?

Reply:
that is just a hanging piece of junk from when i was tinkering around trying to find a better solution to converting a path to a dotted module name. was hoping there was something in ast or various other places but i couldn't find one.
so, i'll remove this line as it's very much not needed.